zosia16/settings/dev.py.example

```diff
@@ -0,0 +1,11 @@
+from .common import *
+
+
+DEBUG = True
+
+ALLOWED_HOSTS = ['*']
+
+SECRET_KEY = 'X'
```

Comment:
>have more meaningful values, or
That doesn't matter in dev-environment.

>Be commented out, with explanation how to obtain proper values, or
How to obtain proper value of secret key. I guess devops guy will know what to do with that, and like i already said, for dev-env it does not matter. Google api key is explained in `README`

>be left out, because these values are set in settings.common.
Based on environment variable. Not too convenient for quick-start.